[Bug] Adapter fails to run in latest edge release

#### Current Behavior
Adapter fails to run.

#### Screenshots/Logs
```
➜   k logs -n meshery meshery-consul-674f759698-8ww2x
time="2021-10-18T19:52:32Z" level=info msg="KUBECONFIG: /home/appuser/.meshery/kubeconfig.yaml" app=consul-adapter
time="2021-10-18T19:52:32Z" level=info msg="Adaptor Listening at port: 10002" app=consul-adapter
time="2021-10-18T19:52:32Z" level=info msg="Registering static workloads..." app=consul-adapter
panic: interface conversion: error is *fs.PathError, not *errors.Error

goroutine 32 [running]:
github.com/layer5io/meshkit/errors.GetCode(0x2d26a20, 0xc000276570, 0x3bf5d78, 0x0)
	/go/pkg/mod/github.com/layer5io/meshkit@v0.2.31/errors/errors.go:90 +0xd1
github.com/layer5io/meshkit/logger.(*Logger).Error(0xc000138178, 0x2d26a20, 0xc000276570)
	/go/pkg/mod/github.com/layer5io/meshkit@v0.2.31/logger/logger.go:57 +0xaa
main.registerCapabilities(0x2a84600, 0x5, 0x2d753c8, 0xc000138178)
	/github.com/layer5io/meshery-consul/main.go:99 +0x266
created by main.main
	/github.com/layer5io/meshery-consul/main.go:83 +0x73a
```

#### Environment

- **Host OS:** Mac 
- **Platform:** Kubernetes
- **Meshery Server Version:** edge-latest
- **Meshery Client Version:** stable-v0.5.66
- **Meshery Adapter Version:** edge-latest


---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

--- a/consul/oam/error.go
+++ b/consul/oam/error.go
@@ -1,0 +1,11 @@
+package oam
+
+import "github.com/layer5io/meshkit/errors"
+
+const (
+	ErrLoadingPathsetCode = "1004"
+)
+
+func ErrLoadingPathset(err error) error {
+	return errors.New(ErrLoadingPathsetCode, errors.Alert, []string{"Could not create a pathset for static component generation"}, []string{err.Error()}, []string{}, []string{})
+}

--- a/consul/oam/register.go
+++ b/consul/oam/register.go
@@ -31,7 +31,7 @@ func RegisterWorkloads(runtime, host string) error {
 
 	pathSets, err := load(workloadPath)
 	if err != nil {
-		return err
+		return ErrLoadingPathset(err)
 	}
 
 	for _, pathSet := range pathSets {
@@ -65,7 +65,7 @@ func RegisterTraits(runtime, host string) error {
 
 	pathSets, err := load(traitPath)
 	if err != nil {
-		return err
+		return ErrLoadingPathset(err)
 	}
 
 	for _, pathSet := range pathSets {

